index.html

```diff
@@ -543,7 +543,7 @@ <h4 id="return-values">Return Value</h4>
     <p>A request initiated by sendBeacon is subject to following properties:</p>
     <ul>
       <li>If the payload is one of <code>application/x-www-form-urlencoded</code>, <code>multipart/form-data</code>, or <code>text/plain</code>, then the request is a simple request that does not require an additional CORS-preflight; same as scripted form-post or XHR/fetch.</li>
```

Comment:
That's not the case. It's the whole reason we changed `sendBeacon()` to use "no-cors". (Hint: redirects.)